ecosystem/sep-0024.md

```diff
@@ -569,6 +638,7 @@ Name | Type | Description
 `deposit_memo_type` | string | (optional) Type for the `deposit_memo`.
 `from` | string | Sent from address, perhaps BTC, IBAN, or bank account.
 `to` | string | Stellar address the deposited assets were sent to.
+`claimable_balance_id` | string | (optional) ClaimableBalanceID is a required parameter when claiming a Claimable Balance operation.
```

Comment:
I found this description confusing. I'd avoid the term parameter since we use the term field in the title. The use of the term required is also confusing in the context of an optional field. And its not clear to me in this context what it means for it to be required when claiming the operation, since the operation isn't claimed. I think we should say what the field contains and when it is used.
```suggestion
`claimable_balance_id` | string | (optional) ID of the Claimable Balance containing the asset, included when an anchor is issuing the asset via a Claimable Balance.
```